app/models/manageiq/providers/vmware/manager_auth_mixin.rb

```diff
@@ -17,7 +18,14 @@ def verify_credentials(auth_type = nil, options = {})
           _log.error("Error Class=#{err.class.name}, Message=#{err.message}")
           raise miq_exception
         end
-      when 'amqp' then verify_amqp_credentials(options)
```

Comment:
Ahhh, it reads better now. Thank you for suggesting this!
The `else` part error message will get prefixed with "Unexpected response returned from system: #{err.message}", which sounds reasonable.